### DISCUSSION TITLE
Toggle checkmarks in view mode

### DISCUSSION THREAD

**Discussion by Akselmo** [gsantner/markor#1529] [Q&A] [answered]
Hi, i use markor a lot for shopping lists

I would like to make the list, toggle view mode and then be able to toggle checklist options on or off in that view

Thank you for the great app!

**Comment by gsantner** [comment, marked as answer]
This has been brought up numerous times, and no this won't be added to Markor.

<https://github.com/gsantner/markor/issues?q=view+mode>
